### PR TITLE
Final mobile portfolio

### PR DESCRIPTION
**Pull request by ahadb802**
**My First Complete Mobile Portfolio Website**
- New cards section
- New About myself Section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -33,20 +33,285 @@ <h2 class="title2">LET’S CONNECT</h2>
...
                     </li>
                     <li>
                         <a href=""><img src="./images/v.svg" alt=""></a>
-                    </li>
-                    
+                    </li> 
                 </ul>
+        </div>
+    </section>
+    <section>
```
Comment (Nyame-Wolf) [inline index.html]: Kindly note that as per the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md), the part with the projects should be one section in a `<section>` tag that contains `<article>` s . The section should be styled with `display: grid` and the individual articles styled using `display:flex`
Currently, your project has the articles split into sections. Please fix this.

Screenshot 2022-10-29 at 10 38 24

File: index.html [hunk 2]
```diff
@@ -33,20 +33,285 @@ <h2 class="title2">LET’S CONNECT</h2>
...
-    
+        
+        </div>
+    </section>
+    <section>
+        <div class="about-my-self">
+        <div class="about-container">
+            <h1 class="about-title">About <br> Myself</h1>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <h2>LET’S CONNECT</h2>
```
Comment (Nyame-Wolf) [inline index.html]: I suggest you change this to a link `<a>` or `<button>`tag. This should redirect users of your page to a contact form or to your social media handles.

File: index.html [hunk 3]
```diff
@@ -33,20 +33,285 @@ <h2 class="title2">LET’S CONNECT</h2>
...
+                </li>
+                <li>
+                    <a href=""><img src="./images/twiter.svg" alt=""></a>
+                </li>
+                <li>
+                    <a href=""><img src="./images/m.png" alt=""></a>
+                </li> 
+            </ul>
+            <Button>Get my resume</BUtton>
+        </div>
```
Comment (Nyame-Wolf) [inline index.html]: I suggest you add equal `paddings and margins` to your project. Currently the left side has more one while the right does not. Its part of the requirements that your project matches figma design
Screenshot 2022-10-29 at 10 36 44

this is your design:
Screenshot 2022-10-29 at 11 12 27

This is the figma design
Screenshot 2022-10-29 at 11 12 03

File: index.html [hunk 4]
```diff
@@ -33,20 +33,285 @@ <h2 class="title2">LET’S CONNECT</h2>
...
+                <li class="couter-image">
+                        <img src="./images/Counter.png" alt="">
+                </li>
+                <li>
+                    <h3 class="year">2015</h3>
+                </li>
+            </ul> 
+        <p class="info-about">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="info-2">
+                <li>
```
Comment (Nyame-Wolf) [inline index.html]: I suggest you add equal `paddings and margins` to your project. Currently the left side has more one while the right does not. Its part of the requirements that your project matches figma design
Screenshot 2022-10-29 at 10 36 44

This is your design
Screenshot 2022-10-29 at 11 18 44

this is the figma design
Screenshot 2022-10-29 at 11 18 51

File: index.html [hunk 5]
```diff
@@ -33,20 +33,285 @@ <h2 class="title2">LET’S CONNECT</h2>
...
+                    <a href=""><img src="./images/v.svg" alt=""></a>
+                </li>
+                <li>
+                    <a href=""><img src="./images/twiter.svg" alt=""></a>
+                </li>
+                <li>
+                    <a href=""><img src="./images/m.png" alt=""></a>
+                </li> 
+            </ul>
+            <Button>Get my resume</BUtton>
```
Comment (Nyame-Wolf) [inline index.html]: I suggest you Implement the button interactions (enable, hover, etc.).
Its a [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) for this milestone
Screenshot 2022-10-29 at 10 36 44

File: index.html [hunk 6]
```diff
@@ -33,20 +33,285 @@ <h2 class="title2">LET’S CONNECT</h2>
...
+                        
+                    
+                </li>
+                <li>
+                    <button class="button-3">Javascript</button>
+                        
+                    
+                </li>
+            </ul>
+                <button class="button-4">See Project</button>
```
Comment (Nyame-Wolf) [inline index.html:87]: I suggest you Implement the button interactions (enable, hover, etc.).
Its a [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) for this milestone
Screenshot 2022-10-29 at 10 36 44

File: index.html [hunk 7]
```diff
@@ -33,20 +33,285 @@ <h2 class="title2">LET’S CONNECT</h2>
...
```
Comment (Nyame-Wolf) [inline index.html:132]: I suggest you Implement the button interactions (enable, hover, etc.).
Its a [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) for this milestone
Screenshot 2022-10-29 at 10 36 44

File: index.html [hunk 8]
```diff
@@ -33,20 +33,285 @@ <h2 class="title2">LET’S CONNECT</h2>
...
```
Comment (Nyame-Wolf) [inline index.html:177]: I suggest you Implement the button interactions (enable, hover, etc.).
Its a [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) for this milestone
Screenshot 2022-10-29 at 10 36 44

File: index.html [hunk 9]
```diff
@@ -33,20 +33,285 @@ <h2 class="title2">LET’S CONNECT</h2>
...
```
Comment (Nyame-Wolf) [inline index.html:222]: I suggest you Implement the button interactions (enable, hover, etc.).
Its a [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) for this milestone
Screenshot 2022-10-29 at 10 36 44

**Review by Nyame-Wolf — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle: 

Hi @ahadb802  :wave: 

Your project page is looking fantastic so far :star_struck: ! 

However, there is still some room for improvement, that can help you have better code and make your project professional :sunglasses: 



## Highlights :sparkles:

- Great work using the right GitHub Flow :100: 
- Linters added and are passing

Congrats on your work so far! :clap:  

## Required Changes ♻️

There are some issues that you still need to work on to go to the next project, but you are almost there! 
_For more information, check the comments under the review :arrow_down:_

## Optional suggestions :ballot_box_with_check: 

_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can improve the quality of your code :100:_

Happy coding and keep it up! :raised_hands: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
  Please, remember to tag me **@Nyame-Wolf**  in your question so I can receive the notification.

  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

  _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html (hunk 1 above)
Comment (ahadb802) [reply index.html]: so their should be 2 sections i.e. work section and about-my-self section?

File: index.html (hunk 4 above)
Comment (ahadb802) [reply index.html]: Screenshot 2022-10-29 183256

-In my preview margins and paddings are set accordingly

**Review by algerina — APPROVED** [review]
Hi @ahadb802 , 
Good job so far! 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous TSEs
 

 ## Optional suggestions
 
 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and to tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**